draft-ietf-quic-transport.md

```diff
@@ -5078,6 +5086,7 @@ Preferred Address {
   IPv4 Port (16),
   IPv6 Address (128),
   IPv6 Port (16),
+  Disable Migration (8),
```

Comment:
We've had a proposal to make this flag per-address.  Need to land this discussion.  If we adopt the suggestion above about Disable Migration means for switching to a different protocol family, this becomes easier to reason about a per-family flag.